packages/studio/src/components/ObjectPropEditor.tsx

```diff
@@ -45,7 +45,7 @@ export default function ObjectPropEditor(props: {
   return (
     <div className={containerClasses}>
       {renderBranchUI(isNested)}
-      <div>
+      <div className="min-w-[112px]">
```

Comment:
just out of curiosity-- what's the goal of this `min-w-[112px]`?

Reply:
It ensures that the undefined menu button is always at least a minimum distance away from the left edge of the right sidebar. This prevents the button from getting partially cut off by going out of bounds when the prop editor would otherwise be very short.

Reply:
I think it looks a little weird to have the ellipses icon for the undefined menu appear a seemingly arbitrary distance to the right of short prop inputs, rather than immediately next to them. I'd rather have the menu be left-aligned with the ellipses icon than move the icon

Reply:
Done